CONTRIBUTING.md

```diff
@@ -1,25 +1,49 @@
+# Guidelines for Contributing to CraftOS-Standards
 
-## Guidelines for Contributing to CraftOS-Standards
-
-Welcome, fellow developer! First off, make yourself familiar with the [tutorial](./Tutorial/GettingStarted.md). It's a long read, so enjoy it with some :coffee:. If you'd like to create new standards, the tutorial should actually give you all that you'll need to know. Thanks for helping us out by standardizing something awesome!
+Welcome, fellow developer! First off, make yourself familiar with the [tutorial](./Tutorial/GettingStarted.md). It's a
+long read, so enjoy it with some :coffee:. If you'd like to create new standards, the tutorial should actually give you
+all that you'll need to know. Thanks for helping us out by standardizing something awesome!
 
 ### A Note on Licensing
-All standards published here on CraftOS-Standards are licensed under the [GNU Free Documentation License](../LICENSE.md). **This includes all code examples**. For the sake of open source software and simplicity alike, libraries for dealing with your standard should also be licensed under [similar FOSS licenses](http://choosealicense.com/). After all, licensing official utilities for an open source standard under a closed source license wouldn't make much sense.
+All standards published here on CraftOS-Standards are licensed under
```

Comment:
Perhaps "licensed" should point to LICENSE.md, and the actual name of the license should point toward its [page on gnu.org](https://www.gnu.org/licenses/fdl-1.3.en.html), for those seeking more information on the license itself.